src/components/pages/AboutPage.js

```diff
@@ -39,6 +81,85 @@ function AboutPage({ classes }) {
         <Typography variant="h3" color="inherit">Simple. Fast. Visual.</Typography>
         <Typography variant="h5" className={classes.heroSubtitle}>Course planning for Northwestern University.</Typography>
       </div>
+
+      <div className={classes.section}>
+        <Typography variant="h3" color="primary" className={classes.header}>Serif.nu redefines your scheduling experience.</Typography>
+        <div className={classes.multipleColumns}>
+          <div>
+            <Typography variant="h5" className={classes.blurbs}>No more drawing out your schedules on paper.</Typography>
+            <Typography variant="h5" className={classes.blurbs}>No more clunkly Excel spreadsheets.</Typography>
+            <Typography variant="h5" className={classes.blurbs}>Plan out your next quarter at Northwestern with ease and speed.</Typography>
+          </div>
+          <img src={studentImage} alt="Student with a laptop" className={classes.studentImage} />
+        </div>
+      </div>
+
+      <div className={classes.sectionWithColor}>
+        <Typography variant="h3" color="primary" className={classes.header}>
+          Features
```

Comment:
Let's take this section out for now, as discussed

Reply:
Removed.